frontend/public/components/factory/modal.tsx

```diff
@@ -10,6 +10,7 @@ import { ActionGroup, Button } from '@patternfly/react-core';
 import store from '../../redux';
 import { ButtonBar } from '../utils/button-bar';
 import { history } from '../utils/router';
+import { CloseButton } from '../utils';
```

Comment:
@nemesis09  This is causing a cyclic dependency and thus causing all those tests to fail.
```suggestion
import { CloseButton } from '../utils/close-button';
```

Reply:
fixed